epytope/EpitopePrediction/ANN.py

```diff
@@ -630,6 +533,19 @@ def _represent(self, allele):
             else:
                 return "%s-%s%s:%s" % (allele.organism, allele.locus, allele.supertype, allele.subtype)
 
+        def convert_alleles(self, alleles):
```

Comment:
Not sure why this is shown as change...

Reply:
I added this function to the class, because it was missing. Same goes for mhcflurry 1.4.3.
I thought it is just consistency. We can also discuss at a later point to make this function and `_represent` as abstract methods